### PR TITLE
Making the lib work with rxjava 1.2.1+ and updated play services

### PR DESCRIPTION
**Pull request by CDehning**

### CODE REVIEW

File: servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java [hunk 1]
```diff
@@ -45,11 +45,11 @@ private GoogleApiClientObservable(Context context, GoogleApi googleApi) {
 
     static Observable<GoogleApiClient> create(Context context, GoogleApi googleApi) {
         return Observable.fromEmitter(new GoogleApiClientObservable(context, googleApi),
```
Comment (Mauin) [inline servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java]: As `Observable.fromEmitter(Action1, BackpressureMode)` has been deprecated with RxJava v1.2.7 this should use `Observable.create(Action1, BackpressureMode)` instead. (Just change `fromEmitter` to `create` and that should work)

**Review by Mauin — CHANGES_REQUESTED** [review]

File: servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java (hunk 1 above)
Comment (CDehning) [reply servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java]: Done

**Review by Mauin — APPROVED** [review]

File: servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java (hunk 1 above)
Comment (Mauin) [reply servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java]: Awesome, thanks! Will wait for CI to go green and will look at building a new release later this week.

**Comment by Rainer-Lang** [comment]
@Mauin Is it possible to get a new release?

**Comment by Mauin** [comment]
@Rainer-Lang I'm pushing the new releases right now.

**Comment by Rainer-Lang** [comment]
Thanks

**Pull request merged**